doc/_sphinx/extensions/flutter_app.py

```diff
@@ -44,13 +44,16 @@ class FlutterAppDirective(SphinxDirective):
         with the matching name.
 
       :show: - a list of one or more run modes, which could include "widget",
-        "popup", and "code". Each of these modes produces a different output:
+        "popup", "code", and "infobox". Each of these modes produces a different
+        output:
           "widget" - an iframe shown directly inside the docs page;
           "popup" - a [Run] button which opens the app to (almost) fullscreen;
           "code" - a [Code] button which opens a popup with the code that was
               compiled.
+          "infobox" - the content will be displayed as an infobox floating on
```

Comment:
This changes of the docs should be extract in a separate PR

Reply:
Done: see #1771